### PR TITLE
alert aojea on kind failures

### PR DESCRIPTION
**Pull request by aojea**
receive alerts on kind jobs failures

### CODE REVIEW

File: testgrid/config.yaml [hunk 1]
```diff
@@ -2328,37 +2328,37 @@ dashboards:
     description: Runs conformance tests using kubetest against latest kubernetes master with a kubernetes-in-docker cluster
     test_group_name: ci-kubernetes-kind-conformance
     alert_options:
-      alert_mail_to_addresses: bentheelder@google.com
+      alert_mail_to_addresses: bentheelder@google.com, antonio.ojea.garcia@gmail.com
```
Comment (BenTheElder) [inline testgrid/config.yaml]: I don't think testgrid expects a space here

Comment (aojea) [reply testgrid/config.yaml]: hehe, python habits

**Comment by BenTheElder** [comment]
needs a rebase

**Comment by Katharine** [comment]
This configuration now lives [in the prowjobs](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/kind/kind.yaml).

**Pull request merged**